### DISCUSSION TITLE
Blokovanie na bežných stránkach

### DISCUSSION THREAD

**Discussion by SiSiROYAL7566** [community/community#145901] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Ahoj,
som obyčajný človek s problémami, ktoré si pomáham riešiť na webe no blokujú sa mi obyčajné stránky, Chrome takmer nemôžem používať. Nič sa nedá načítať, zdá sa mi akoby niekto, alebo niečo bránilo v mojom patraní po spravodlivosti. DávnejšiecSte ma sem k Vám pozvali, najprv cez privacywall ešte keď nebola verejná ja nemám IT vzdelanie, som iba Bc.Histórie. Ak mi pomôžete budem vďačná, som totiž aj chorá invalidná dôchodkyňa. Nemôžem si dovoliť pkatiť stránky. Tak ďakujem s pozdravom Silvinka

**Comment by mecodeatlas** [comment]
Hi @SiSiROYAL7566, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
